wagtail/admin/views/icons.py

```diff
@@ -0,0 +1,8 @@
+from django.shortcuts import render
+from django.views.decorators.cache import cache_control
+
+
+@cache_control(max_age=3600, private=True)
+def sprite(request):
```

Comment:
Is this actually used anywhere? I don’t think I’ve seen it.

Reply:
I dropped it.